lambda/jvm/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -53,6 +53,7 @@ private[lambda] abstract class IOLambdaPlatform[Event, Result]
               val json = result.asJson
               val writer = new OutputStreamWriter(output)
               Printer.noSpaces.unsafePrintToAppendable(json, writer)
+              writer.close()
```

Comment:
Should this be wrapped in a `Resource` so it's closed even if an exception occurs?

Reply:
Hmm, good point. Maybe. I didn't reach for `Resource` here because I was stupidly focused on optimizing into a single `IO { ... }` 😅 

I'm not sure if it makes a difference in practice, since if we get an error at this stage the Lambda has effectively crashed since there is nothing to catch it. But I do think it's worth making this code idiomatic.